CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
 
 
-find_package(IRODS 4.2.6 EXACT REQUIRED)
+find_package(IRODS 4.2.8 EXACT REQUIRED)
 set(IRODS_PLUGIN_VERSION_MAJOR 2)
 set(IRODS_PLUGIN_VERSION_MINOR 5)
```

Comment:
Should this not also be updated in line with irods minor version?

Reply:
needs to be converted to use the four digit version numbers.  can discuss whether 4.2.7.0 is best/possible before 4.2.8.0.